src/containers/WorkspaceView.tsx

```diff
@@ -1,50 +1,184 @@
-import { Button } from '@material-ui/core';
-import React, { FunctionComponent, useCallback, useState } from 'react';
-import Splitter from '../extensions/common/Splitter';
-import { Recording, RecordingInfo } from '../reducers/recordings';
+import React, { FunctionComponent, useMemo } from 'react';
```

Comment:
This is the main workspace view, which is essentially the home page for labbox-ephys, which handles the routing between RecordingsView, RecordingView, SortingView, etc.

Reply:
Gotcha. It feels a little weird to me to have this level of routing logic in what's nominally a view component (but it's also not like I have a better suggestion...)